public-api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -204,359 +204,21 @@ FROM  priorstats p
 join overallstats o on p.stratum1_id = o.stratum1_id and p.stratum2_id = o.stratum2_id
 group by o.stratum1_id, o.stratum2_id, o.total, o.min_value, o.max_value, o.avg_value, o.stdev_value"
 
-# 1816 Measurement response by age at occurrence distribution
```

Comment:
This data is not being used at this point of time. So removing the queries, saved on local system for reference.